pages/api/webhooks/checkout-calculate-taxes.ts

```diff
@@ -21,15 +22,24 @@ const handler: NextApiHandler = async (request, response) => {
       .json({ success: false, message: error.message });
     return;
   }
+  const saleorDomain = request.headers[SALEOR_DOMAIN_HEADER];
```

Comment:
Is it always set? What if it was not?

Reply:
You can use middlewares from app-sdk for checking if the header is present: https://github.com/saleor/saleor-app-sdk/blob/main/src/middleware.ts#L18

Reply:
As @witoszekdev  said. All basic Saleor's data, like domain, event, signature etc is checked in app-sdk. If missing, then we will immediately return error

Reply:
Yes, but you're not using app-sdk for **this** handler 😅

https://github.com/saleor/saleor-taxjar/blob/a933762649a67b7740d54a49c76f2b76ab943205/pages/api/webhooks/checkout-calculate-taxes.ts#L50

Reply:
> You can use middlewares from app-sdk for checking if the header is present: https://github.com/saleor/saleor-app-sdk/blob/main/src/middleware.ts#L18

@witoszekdev  I am currently working on using all middlewares from app-sdk, but this is ongoing as I have some issues that need to be clarified with app-sdk team (and it is ongoing)